dotcom-rendering/src/web/lib/decideContainerOverrides.ts

```diff
@@ -44,7 +44,27 @@ const textCardKicker = (containerPalette: DCRContainerPalette): string => {
 };
 
 const textCardByline = textCardKicker;
-const textContainerDate = textCardKicker;
+
+const textContainerDate = (containerPalette: DCRContainerPalette): string => {
+	switch (containerPalette) {
+		case 'LongRunningPalette':
+			return '#c70000';
+		case 'LongRunningAltPalette':
+			return '#8b0000';
+		case 'SombrePalette':
+			return '#c1d8fc';
+		case 'SombreAltPalette':
+			return '#ff5943';
+		case 'InvestigationPalette':
+			return '#ffe500';
+		case 'BreakingPalette':
+			return '#8b0000';
+		case 'EventPalette':
+			return '#c70000';
+		case 'EventAltPalette':
+			return '#c70000';
```

Comment:
Do you this switch statement could be order alphabetically?

Reply:
I’ve investigated linting things before, but it’s a big endeavour. For another PR.